src/transformers/pipelines/feature_extraction.py

```diff
@@ -79,4 +79,10 @@ def __call__(self, *args, **kwargs):
         Return:
             A nested list of :obj:`float`: The features computed by the model.
         """
-        return super().__call__(*args, **kwargs).tolist()
+        results = super().__call__(*args, **kwargs)
+        if hasattr(results, "tolist"):
+            results = results.tolist()
+        else:
+            print("Sequential results")
```

Comment:
This should use the internal logger insread.

Reply:
I'll remove it.